pmoves/services/creator-operator/tests/test_fanout.py

```
@@ -0,0 +1,43 @@
+    assert sinks.nats and sinks.nats[0][0] == "creator.operator.result.v1"
+    assert sinks.notebook and sinks.notebook[0] == VALID_RESULT["transcript"]
+    assert sinks.discord and "seed" in sinks.discord[0][0]
+    assert sinks.n8n and sinks.n8n[0]["nodes"][0]["parameters"]["workorder_id"] == "wo_test1"
+
+
+def test_emit_result_validates_before_fanout():
+    sinks = FakeSinks()
+    bad = dict(VALID_RESULT, status="maybe")
+    try:
+        asyncio.run(emit_result(bad, VALID_WORKORDER, sinks, model_id="x", license="y"))
+        assert False, "should have raised on invalid result"
```

Ruff (0.15.15)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)